docs/testnet/essentials.md

```diff
@@ -1,22 +1,20 @@
-# Essential Testnet Information
-An easy-to-read list of essential parameters and configuration settings for Obscuro Testnet.
+# Essential Information
+Essential information, parameters and configuration settings for the Oikos testnet.
+
+## Limitations on Oikos
+These are the key limitations to be aware of when developing for the Oikos testnet:
+
+1. New nodes cannot be added. For now the number of Obscuro nodes is fixed.
+1. Data revelation is not implemented yet.
+1. Security is not fully implemented. Some keys are still hardcoded.
+1. The decentralised bridge is limited to two hardcoded ERC20 tokens.
+1. The Layer 1 is currently a hosted network. In the next iteration we'll connect Obscuro to Goerli.
+1. The "Wallet Extension" is not fully polished yet.
 
 ## Connection to an Obscuro Node
 - **RPC http address:** `testnet.obscu.ro:13000`
 - **RPC websocket address:** `testnet.obscu.ro:13001`
 
-## Custom Network for MetaMask
```

Comment:
How come we're removing this and the ObscuroScan URL?

Reply:
They've been moved e.g. ObscuroScan is now accessed directly from the menu under tools